wagtailio/static/sass/components/_quotes.scss

```diff
@@ -0,0 +1,72 @@
+.quotes {
+    &__header {
+        color: $color--off-black;
+        padding-bottom: 30px;
+        grid-column: 2 / span 4;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 7;
```

Comment:
Suggestion: Reduce this to `grid-column: 2 / span 4` so a long title won't overlap the quote icon